### DISCUSSION TITLE
Example dataset

### DISCUSSION THREAD

**Discussion by Hamidm83** [alicevision/AliceVision#1058] [Q&A] [answered]
Hi, 
I like to get familiar with AliceVision but I couldn't find any dataset example. I am especially interested to know the structure of sfm/JSOM file format and how to read image metadata in a folder. 

Thanks, 
Hamid

**Comment by natowi** [comment, marked as answer]
You can run a reconstruction with the https://github.com/alicevision/dataset_monstree dataset and read the sfm/JSON files.
I started to document the format, but I need to remember where I put it. https://github.com/alicevision/meshroom/issues/683 has some information
Here are some of my personal notes https://gist.github.com/natowi/ad9ce6d9f912e089bba9b69ec91d7115 (may be partially outdated)
